application/github/v0/config/definition.json

```diff
@@ -0,0 +1,27 @@
+{
+  "availableTasks": [
+    "TASK_GET_ALL_PULL_REQUESTS",
+    "TASK_GET_PULL_REQUEST",
+    "TASK_GET_COMMIT",
+    "TASK_GET_REVIEW_COMMENTS",
+    "TASK_CREATE_REVIEW_COMMENT",
+    "TASK_GET_ALL_ISSUES",
+    "TASK_GET_ISSUE",
```

Comment:
I make tasks to get a specific pr/issue to simplify the response. If the client can only use multiple ones, they may need to use another component to filter out the response they want and make the pipeline more complex. 

Moreover, some detailed information may not be provided in the multiple ones' responses. The purpose of build "get all" here is to provide a tool for users to know what pr/issue the repo has if they don't know the pr/issue number.

Reply:
Maybe we can combine the two multiple tasks into a "TASK_GET_REPO_INFO" and return all the info users might want to know?

Reply:
> Maybe we can combine the two multiple tasks into a "TASK_GET_REPO_INFO" and return all the info users might want to know?

I think your original way will be fine!
Aligning the vendors could bring as benefit in terms of workflow use case in my opinion.

It may be too various to know what users want to know in the cases of workflow because every company has different flows even when there are best practices.

Reply:
Got it!

One more thing, there are page, page_number options to determine how many items would be returned, should I include these in the field?